features/step_definitions/common_steps.rb

```diff
@@ -125,7 +125,7 @@
 Then(/^Claim '(.*?)' should be listed with a status of '(.*?)'(?: and a claimed amount of '(.*?)')?$/) do |case_number, status, claimed|
   my_claim = @external_user_home_page.claim_for(case_number)
   expect(my_claim).not_to be_nil
-  expect(my_claim.state.text).to eq(status.upcase)
+  expect(my_claim.state.text).to include(status.upcase)
```

Comment:
has it changed to return an array?

Reply:
@naseberry

Reply:
No it hasn't, this should not have been committed, therefore I've now removed it.